### PR TITLE
Update to org.gnome.Platform//45

### PR DESCRIPTION
**Pull request by vdualb**
Closes https://github.com/flathub/org.mypaint.MyPaint/issues/14
Closes https://github.com/flathub/org.mypaint.MyPaint/pull/13

### CODE REVIEW

File: org.mypaint.MyPaint.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "org.mypaint.MyPaint",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "42",
+    "runtime-version": "44",
```
Comment (bbhtt) [inline org.mypaint.MyPaint.json:4]: Can you bump it to 45?

**Comment by RocketRide9** [comment]
bot, build

**Comment by RocketRide9** [comment]
> No sure how the no change rebuild will solve `ModuleNotFoundError: No module named 'packaging'`

i thought first build was done for my first commit where i only changed python3-modules file and second build for commit which includes platform version bump failed for internal issue.

**Comment by bbhtt** [comment]
Copy paste from here https://github.com/flathub/io.github.hydrusnetwork.hydrus/commit/127e2ea7f48e66ef21c3cab3aa4d8a622d085923, the way you did it, openblas will probably take a long time to build. You need to set the build-env for numpy to point it to blas in /app/lib.

pyproject-metadata is needed for numpy 1.26.0

**Comment by bbhtt** [comment]
Argh, mypaint needs a patch because python 3.12. rU needs to be changed to U, I can look into it tomorrow if you want.

**Comment by RocketRide9** [comment]
> Argh, mypaint needs a patch because python of 3.12. rU needs to be changed to U, I can look into it tomorrow if you want.

are you sure? in this upstream commit they removed 'U': https://github.com/mypaint/mypaint/commit/032a155b72f2b021f66a994050d83f07342d04af . yes please, look into it. i will squash commits in a bit. i dont want anyone to see this commit history =)

**Comment by RocketRide9** [comment]
bot, cancel build

**Comment by bbhtt** [comment]
> are you sure?

Oh sorry I said the opposite, yes part of that is what we need.

**Comment by bbhtt** [comment]
I split your commits so that the history is a bit clearer and added some cleanups. Let me know if it works.

**Comment by RocketRide9** [comment]
seem to work. thanks.

**Comment by bbhtt** [comment]
Thanks, let's merge.

**Pull request merged**